openapi-to-har.js

```diff
@@ -19,6 +19,7 @@
  * }
  */
 const OpenAPISampler = require('openapi-sampler');
+const QueryString = require('querystring');
```

Comment:
I think using this library is problematic for two reasons:

1. `querystring` is part of the Node.js standard library, and thus not available in the browser. However, openapi-snippet should remain usable in the browser.
2. Even within Node.js, `querystring` is actually marked as "legacy" (see https://nodejs.org/api/querystring.html).

Depending on what's needed from this module, we might have to introduce a new dependency, for example for https://github.com/sindresorhus/query-string.